rest-api/api/base_api.py

```diff
@@ -61,6 +116,7 @@ def get(self, id_=None, participant_id=None):
       if participant_id != obj.participantId:
         raise NotFound("%s with ID %s is not for participant with ID %s" %
                        (self.dao.model_type.__name__, id_, participant_id))
+    self._save_raw_request(obj)
```

Comment:
Could we just put this call in _make_response once ? Seems like all the verbs call it.

Reply:
Its possible that not all API's call `super()` in their `_make_response` override.  This way makes sure `_save_raw_request()` is always called, and is future proof even if all child implementations call the `super()` method now.